### DISCUSSION TITLE
Unable to download with the new UI

### DISCUSSION THREAD

**Discussion by Krachwayne** [aclap-dev/video-downloadhelper#1357] [Help and Q&A]
### Your question

Hello I've ran into a issue the past couple of weeks where I've tried to download sites off of pretty much anywhere and all of them give me the same error message below When using the new UI(Goes back to working when using the old UI)
 
Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!).

### Addon & CoApp version and details

version: 9.1.0.44
channel: stable
build date: 2024-09-04
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): d0d7650f6f4b4d34
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\DBZfa",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
I don't think that has anything to do with it, give an example or several where it happens.
